File: ApiSuvesaPos/Negocio/Modelo/inventario/prestamos/prestamo.vb
Namespace Modelo
    Public Class prestamo
        Public Property ID As Integer
        Public Property fecha As DateTime
        Public Property estado As Boolean
        Public Property anulado As Boolean
        Public Property Entrada As Boolean
        Public Property Salida As Boolean
        Public Property observacion As String
        Public Property nombre_destino As String
        Public Property boleta As String
        Public Property destinatario As Integer
        Public Property nombre_destinatario As String
        Public Property destino As Integer
        Public Property transportista As String
        Public Property Id_UsuariCreo As String
        Public Property BoletaProveedor As String
        Public Property Detalle_Prestamo As New List(Of detalle_prestamo)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/prestamos/prestamo.cs
namespace Modelo
{
    public class prestamo
    {
        public int ID { get; set; }
        //public DateTime fecha { get; set; }
        public bool estado { get; set; }
        public bool anulado { get; set; }
        public bool Entrada { get; set; }
        public bool Salida { get; set; }
        public string observacion { get; set; }
        public string nombre_destino { get; set; }
        public string boleta { get; set; }
        public int destinatario { get; set; }
        public string nombre_destinatario { get; set; }
        public int destino { get; set; }
        public string transportista { get; set; }
        public string Id_UsuariCreo { get; set; }
        public string BoletaProveedor { get; set; }
        //public List<detalle_prestamo> Detalle_Prestamo { get; set; } = new List<detalle_prestamo>();
    }
}
